beta/src/pages/learn/thinking-in-react.md

```diff
@@ -25,45 +26,47 @@ The JSON API returns some data that looks like this:
 ]
 ```
 
-The mockup looks like this:
+मॉकअप इस तरह दिखता है:
 
 <img src="/images/docs/s_thinking-in-react_ui.png" width="300" style={{margin: '0 auto'}} />
 
-To implement a UI in React, you will usually follow the same five steps.
+React  में  UI  को लागू करने के लिए, आप आमतौर पर एक ही पांच चरणों का पालन करेंगे।
+
+## Step 1: एक कुम्पोनेंट पदानुक्रम में UI को तोड़ो {/*step-1-break-the-ui-into-a-कौम्पोनॅन्ट-hierarchy*/}
```

Comment:
`{/*step-1-break-the-ui-into-a-कौम्पोनॅन्ट-hierarchy*/}` => `{/*step-1-break-the-ui-into-a-component-hierarchy*/}`

lets not translate these